src/sentry/scripts/digests/digests.lua

```diff
@@ -9,22 +19,85 @@ function table.extend(t, items, length)
     end
 end
 
-function table.slice(t, start, stop)
-    if stop == nil then
-        stop = #t
+local function chunked(size, iterator, state, ...)
```

Comment:
No, unfortunately. I'd love to figure out a solution for this — the best thing that I've been able to come up with so far has been to build a `TEST` command that runs it's own suite. I guess that might be better than nothing? 

Ideally we'd be able to have a test script/runner that:

- exists outside of the script that we're actually shipping to the server (I haven't figured out a pattern for doing this that makes sense)
- can use an assertion library (the stock Lua assertions are pretty weak),
- can run multiple tests with teardown that flushes the test data
- I think I had other ideas/requirements before but I haven't thought about them in a while